frontend/src/components/Input/index.module.css

```diff
@@ -0,0 +1,54 @@
+.input {
+  padding: 20px 24px;
+  font-family: "Work Sans", sans-serif;
+  font-size: 16px;
+  font-style: normal;
+  font-weight: 400;
+  line-height: 20px;
+  color: #000;
+  border: 1px solid #9f18d9;
+  border-radius: 10px;
+  box-shadow: 0 4px 5px #9f18d9 0.5;
+}
+
+.input:active {
+  color: #000;
+  outline-style: none;
+}
+
+.input:focus {
+  outline-style: none;
+}
+
+.input:disabled {
+  color: #404040;
+  background: #bfbfbf;
+  border: 1px solid #404040;
+  border-radius: 10px;
+  box-shadow: 0 4px 5px #808080;
+}
+
+.inputError {
+  border-color: red;
+}
+
+.container {
+  display: inline-grid;
+  font-size: 12px;
+  line-height: 16px;
+  color: #F73F52;
+}
```

Comment:
Let's place styles in the order they appear in the HTML. So, in this case, `container` would be a the top. Unless you go with the happy path first. Also, I'd rename this to `errorContainer` or something because when I see `container` I'm expecting a root element.

Reply:
I might even consider creating an `Error` component

Reply:
I fixed the first topic. Should I change this:
```
<div className={!error ? "" : styles.errorContainer}>
      {!error ? "" : <span>{errorMessage}</span>}
      <input
        disabled={disabled}
        type={type}
        placeholder={placeholder}
        className={
          !error ? styles.input : `${styles.input} ${styles.inputError}`
        }
      />
</div>
```
to an `Error` component?